Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -494,15 +496,15 @@ class StatusTableViewController: UITableViewController, UIGestureRecognizerDeleg
             case .Date:
                 cell.textLabel?.text = NSLocalizedString("Last Sentry", comment: "The title of the cell containing the last updated date")
 
-                if let date = dataManager.latestPumpStatus?.pumpDateComponents.date {
+                if let date = dataManager.latestPumpStatusDate {
```

Comment:
I'm OK with it either way; I'm planning to kill off all of these text lines in favor of something more glance-able, and I would probably use `latestPumpStatusDate`